pkg/services/sit_address_update.go

```diff
@@ -1,13 +1,34 @@
 package services
 
 import (
+	"github.com/gofrs/uuid"
+
 	"github.com/transcom/mymove/pkg/appcontext"
 	"github.com/transcom/mymove/pkg/models"
 )
 
-// ApprovedSITAddressUpdateCreator Interface for the service object that creates a SIT Address Update
+// ApprovedSITAddressUpdateCreator Interface for the service object that creates a approved SIT Address Update with a distance < 50 miles
```

Comment:
```suggestion
// ApprovedSITAddressUpdateCreator Interface for the service object that creates an approved SIT Address Update
```
This function can be used to create an update with a distance > 50 miles if the TOO creates it.